### DISCUSSION TITLE
Saving calendar entries over a defined period of time

### DISCUSSION THREAD

**Discussion by 011V32** [mampfes/hacs_waste_collection_schedule#1840] [Q&A]
Hello.
Unfortunately, the calendar entries from the past few days are always deleted immediately.
(I assume this is because the entries are no longer available when they get retrieved.)
But I want to check every now and then to see when the trash was last picked up.
Is there a way to save the entries over a certain period of time?
Thanks.

**Comment by 5ila5** [comment]
There is currently no way to store events that are no longer provided by the source. The source probably does not get the dates anymore, and all events are erased if new ones are fetched. I can take a look if the is way to get the past dates for your source if you say which one you're using.

**Reply by 011V32** [reply]
OK. Thx.
The source would be umweltprofis.at, but they didn‘t have that possibility.

**Reply by nuki47** [reply]
Does the integration for umweltprofis.at work for you?

**Reply by 011V32** [reply]
> Does the integration for umweltprofis.at work for you?

Yes 👍

**Reply by nuki47** [reply]
> > Does the integration for umweltprofis.at work for you?
> 
> Yes 👍

ok, configuration direct in config.yaml or in the integrations GUI? In which area do you live?

**Reply by 011V32** [reply]
> ok, configuration direct in config.yaml or in the integrations GUI? In which area do you live?

Back then with Yaml, since the GUI support in the GUI.

**Reply by nuki47** [reply]
Can you share your config please?

**Reply by 011V32** [reply]
> Can you share your config please?

Old Yaml config:
![image](https://github.com/user-attachments/assets/6067b617-1be7-4354-bea9-781e88081248)
![image](https://github.com/user-attachments/assets/2b62783f-649b-48c1-bd7d-9c3e66f9aa3c)

In the GUI you couldn't see that much...
![image](https://github.com/user-attachments/assets/fa616d00-f125-472a-af97-855708c67064)
![image](https://github.com/user-attachments/assets/a8b213a2-c3b8-4297-a8b6-a9480d080145)

**Reply by nuki47** [reply]
How do I get for the old yaml config the full xmurl?
https://data.umweltprofis.at/opendata/

**Reply by 5ila5** [reply]
@nuki47 as described [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/data_umweltprofis_at.md), you need to get the url from https://data.umweltprofis.at/opendata/AppointmentService/index.aspx

**Comment by nuki47** [comment]
Thank, but in the GUI I got this error

<img width="254" alt="image" src="https://github.com/user-attachments/assets/9b1808b3-5e17-402f-94d9-c68d075aa3bc">



Am Mo., 28. Okt. 2024 um 10:48 Uhr schrieb 011V32 ***@***.***

**Reply by 5ila5** [reply]
Maybe take a look in your logs, this might be related to #2907
